src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You should provide a list of fonts, separated by commas, where the next font is used if the previous one is not available. For example, you can use a generic font family name as a fallback, like `sans-serif` or `serif`.